test/spec/properties.js

```diff
@@ -455,4 +467,140 @@ describe('properties', function() {
 
   });
 
+  describe('multiple inherited properties', function() {
+
+    // const createModel = createModelBuilder('test/fixtures/model/');
+    var mhModel = createModel([
+      'properties',
+      'multiple-inherited-properties'
+    ]);
+
+    describe('descriptor', function() {
+
+      it('should provide type', function() {
+
+        // when
+        var Type = mhModel.getType('mh:MultipleInherited');
+        var descriptor = model.getElementDescriptor(Type);
+
+        // then
+        expect(Type).to.exist;
+        expect(descriptor).to.exist;
+        expect(descriptor.propertiesByName.any).to.exist;
+        expect(descriptor.propertiesByName['mh:any']).to.exist;
```

Comment:
TODO: Verify which descriptor is equal (strict equality) to which descriptor.

`expect(descriptor.propertiesByName.any).to.equal(descriptor.propertiesByName['mh:any'])`

Reply:
TODO: Add test to verify the following:

* Package a defines `a:Foo#a`
* Package b defines `b:Other#a`
* Package c defines `c:MultipleInherited { a:Foo, b:Other }`
* :arrow_right: What is the local descriptor for `c.MultipleInherited#a`?

Reply:
> TODO: Verify which descriptor is equal (strict equality) to which descriptor.
>
> `expect(descriptor.propertiesByName.any).to.equal(descriptor.propertiesByName['mh:any'])``

Another hit! [Added the test](https://github.com/bpmn-io/moddle/pull/39/commits/1af0602cfb11db33cac52c66a56fa2369e90fc68) (using `eql()` for deep-equal) . The error is that`any` describes `props:any` instead of the local defined 'any' a.k.a. 'mh:any'. I only inspected `descriptor.properties` and this is ok ...
[Fixed](https://github.com/bpmn-io/moddle/pull/39/commits/36aaaccc0c6ac879ef16199786e24d21aa45a74e).

Reply:
> TODO: Add test to verify the following:
> 
>     * Package a defines `a:Foo#a`
> 
>     * Package b defines `b:Other#a`
> 
>     * Package c defines `c:MultipleInherited { a:Foo, b:Other }`
> 
>     * arrow_right What is the local descriptor for `c.MultipleInherited#a`?

I hope I got you right with [this](https://github.com/bpmn-io/moddle/pull/39/commits/d9edaf255fd398455c8ffccf9f47137b739f1437) commit, but maybe I miss your point as it seems to be redundant.